src/condor_startd.V6/command.cpp

```diff
@@ -1696,6 +1698,13 @@ activate_claim( Resource* rip, Stream* stream )
 
 	rip->dprintf( D_FULLDEBUG, "Read request ad and starter from shadow.\n" );
 
+	// if request has a flags that control the reply, look them up and then
+	// delete them from the request ad
+	if (req_classad->Lookup(ATTR_send_failure_ad)) {
+		req_classad->LookupBool(ATTR_send_failure_ad, send_failure_ad);
```

Comment:
You can just put the LookupBool() in the conditional.

Reply:
TJ says that he wants to distinquish between the existance of the attribute and what what it has.